test_common.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-

```

Comment:
Why is this not in `sklearn/metrics/cluster/tests`?

Where it is, the tests are not being run by `make test` or the continuous integration servers.